server/user.go

```diff
@@ -37,3 +37,35 @@ func getOrCreateUser(ctx context.Context, sub string, info *oidc.UserInfo) (*Use
 	}
 	return &user, nil
 }
+
+type GetMeOutput struct {
+	Body User
```

Comment:
might be the first time we expose it in the API, we can set the fields of the struct to have nicer names when serialized in json.